qiskit/pulse/builder.py

```diff
@@ -2121,7 +2278,7 @@ def barrier(*channels_or_qubits: Union[chans.Channel, int], name: Optional[str]
     in the case where we are calling an outside circuit or schedule and
     want to align a pulse at the end of one call:
 
-    .. jupyter-execute::
+    .. code-block::
 
         import math
 
```

Comment:
Yeah, I did this on purpose because this section of the docs read more like a notebook to me and since we're not actually executing anything I left it as is so the rendering was the same.